src/init-ceph.in

```diff
@@ -463,6 +475,7 @@ for name in $what; do
 	    if daemon_is_running $name ceph-$type $id $pid_file; then
 		printf "$name: running "
 		do_cmd "$BINDIR/ceph --admin-daemon $asok version 2>/dev/null" || echo unknown
+		[ `uname` = FreeBSD ] && echo
```

Comment:
@wjwithagen neither does my `bin/init-ceph status` has `\n` at the end of daemon status line
```
$ bin/init-ceph status
=== mon.a ===
mon.a: running {"version":"Development"}=== mon.b ===
mon.b: running {"version":"Development"}=== mon.c ===
mon.c: running {"version":"Development"}=== osd.0 ===
osd.0: not running.
=== osd.1 ===
osd.1: running {"version":"Development"}=== osd.2 ===
osd.2: not running.
```

i just posted #13351 to address this problem. so probably you can drop this change?